### PR TITLE
Added the connector-odoo2odoo repo

### PR DESCRIPTION
**Pull request by elicoidal**

### CODE REVIEW

File: tools/oca_projects.py.save [hunk 1]
```diff
@@ -0,0 +1,170 @@
+# -*- coding: utf-8 -*-
```
Comment (pedrobaeza) [inline tools/oca_projects.py.save]: You have to remove this backup file

Comment (elicoidal) [inline tools/oca_projects.py.save]: OK: shall I remove and then commit again?

Comment (pedrobaeza) [inline tools/oca_projects.py.save]: Remove it, and make:
```
git add --all
git commit --amend
```
This way, you won't have an extra commit.

Comment (elicoidal) [inline tools/oca_projects.py.save]: great: done

Comment (pedrobaeza) [inline tools/oca_projects.py.save]: I'm afraid it's still there. When pushing, you have to make git push -f

Comment (elicoidal) [inline tools/oca_projects.py.save]: Not sure I didnot do something stupid...

Comment (elicoidal) [inline tools/oca_projects.py.save]: Actually I did... I pushed on the main branch instead of mine
Sorry about that.

Comment (pedrobaeza) [inline tools/oca_projects.py.save]: OK, the branch contains now correct things, so let's stay this way.

**Comment by pedrobaeza** [comment]
@elicoidal, you have to push again your changes, because I can't retrieve then after restoring properly the branch. I have also protected the branch against force pushing to avoid this kind of incidents again.

**Comment by elicoidal** [comment]
@pedrobaeza OK I will. Another option is to remove my super-powers.
I dont need them and I dont know how to use them...

**Comment by pedrobaeza** [comment]
"A great power means a great responsability"  :wink:

No, there's no problem now because the "super-powers" also allow you to merge some branches when needed, but now the main branch is protected to avoid this by mistake.

**Comment by elicoidal** [comment]
Done. Thanks for your help and patience!

**Comment by moylop260** [comment]
FYI github has a new validation to avoid `git push -f` in stable branches.
no-push-f

**Comment by pedrobaeza** [comment]
Thanks, @moylop260. I activated it when this came.

**Pull request closed without merging**